src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -71,6 +74,10 @@ class ServerPaginatedTableView extends React.Component {
     });
   }
 
+  changeToPrevDataState = (prevProps) => {
+    this.setState({ data: prevProps.data, count: prevProps.count });
```

Comment:
Do we need this?

Reply:
we need this to make sure that we still have the data while the table is loading, otherwise, when the data is loading, the data value is 'undefined'.